### DISCUSSION TITLE
Bug Pull & Request

### DISCUSSION THREAD

**Discussion by Lukynnnn** [community/community#48842] [Pull Requests]
### Select Topic Area

Bug

### Body

Hi, I am just want to pull and request data what I upload, but I cant, it showing me this: 
![obrazek](https://user-images.githubusercontent.com/59299731/221976572-99b66a5a-c21d-4a69-b4c7-29d9f02c6511.png)

**Comment by vishnubansal001** [comment]
Hi @Lukynnnn bro,
Here are some steps that may help you:

1. Verify that the data has been uploaded: Check that you have successfully uploaded the data to the correct location on the platform. If the data is not present or has not been uploaded, you will not be able to pull or request it.
2. Check the permissions: Ensure that you have the appropriate permissions to access the data. Check with the platform's administrator or the person who uploaded the data to verify that you have the necessary permissions.
3. Check the file format: Make sure that the file format is compatible with the platform you are using. Some platforms may not support certain file formats, which can cause issues with pulling or requesting data.
4. Verify your code: Check your code to make sure that you are using the correct syntax and that there are no errors. Double-check your code to make sure that it matches the platform's API or data retrieval guidelines.
5. Seek help: If you are still having trouble, seek help from the platform's support team or online forums. They may be able to provide additional guidance or troubleshooting steps to help you resolve the issue.

Remember, data retrieval can be complex and may require patience and persistence. Keep trying, and seek help when needed.
